app/services/symphony_client.rb

```diff
@@ -166,6 +166,8 @@ def error_code(response)
       return if response.status.ok?
 
       JSON.parse(response.body).dig('messageList')[0].dig('code')
+    rescue JSON::ParserError
+      nil
```

Comment:
I want to make sure I understand this bit: When JSON tries to parse the response body and then dig for code, if it hits a bad formatting problem or whatever in the JSON itself, it will raise up and set error_code to nil?

Reply:
yup, will return nil for any response without the expected messageList and code fields

Reply:
this would only catch errors during `parse` not during `dig` or `first`... right?

Reply:
I think so yes

Reply:
I wonder if it'd `raise` this if there was a problem, like if dig or first hit a fatal. Anyway, this is good.